Signal/src/view controllers/CallViewController.swift

```diff
@@ -187,15 +81,15 @@ class CallViewController: UIViewController, CallDelegate {
         contactsManager = Environment.getCurrent().contactsManager
         let callService = Environment.getCurrent().callService!
         callUIAdapter = callService.callUIAdapter
-        audioService = CallAudioService()
+        audioService = CallAudioService(handleRinging: callUIAdapter.hasManualRinger)
```

Comment:
I don't understand why CallAudioService is owned by CallVC.  CallVC could be dismissed (so user can message).  Wouldn't we want to keep ringing even though user has left CallVC?